### ISSUE TITLE
Create docker container for running python processing tasks

### ISSUE THREAD

**Issue by patcon** [sidewalklabs/oldto#22]
Any objection? Makes sense to me to have one container for building app and running it  (:ballot_box_with_check:), and another for processing data :)

Happy to hear criticism of that though!

**Comment by danvk** [comment]
Hey @patcon I usually set up a virtualenv and do development outside of Docker to keep things local. No worrying about whether files are mounted in the Docker image and such. Curious to hear if you had trouble installing the requirements or running `make`; if so, that's definitely something we should fix!

**Comment by patcon** [comment]
Yep, I did hit some bumps with make. If I recall, i couldn't get the appropriately version of python installed on my ubuntu setup (and so package compilation was failing), and pyenv was giving me problems. I'd already set up docker (since it's the recommended way to use the app), so that felt done already. I came back after a few days and eventually got pyenv working for another project, but it was a bump that might have lost someone else from participating. When I come back to work on this, I'll try again and see if the issues persist now that pyenv is an option for me

Fwiw, I always find it humbling to walk through project setup with people in person, so that might be a neat use of a hacknight if you had bandwidth. Or alternatively, i could record while someone else new goes through it, and share the screencast :)

Anyhow, thanks for your time!

**Comment by DOsinga** [comment]
Was this issue Python 3.6 on Ubuntu? Do you remember what the scientific packages were that were failing? Maybe we could suggest an anaconda based install that would be easier if more people have this issue.
